lib/snapshot/reports_generator.rb

```diff
@@ -13,16 +13,20 @@ def generate
       Dir[File.join(screens_path, "*")].sort.each do |language_folder|
         language = File.basename(language_folder)
         Dir[File.join(language_folder, '*.png')].sort.each do |screenshot|
-          available_devices.each do |key_name, output_name|
-            next unless File.basename(screenshot).include?(key_name)
+       	   ["portrait", "landscape"].each do |orientation|	
+	          available_devices.each do |key_name, output_name|
+    	        next unless File.basename(screenshot).include?key_name
+				  if File.basename(screenshot).include?orientation
```

Comment:
Put space between the method name and the first argument.<br>Inconsistent indentation detected.<br>Tab detected.